public/app/features/dashboard/components/TransformationsEditor/TransformationsEditor.tsx

```diff
@@ -40,6 +42,7 @@ interface TransformationsEditorProps extends Themeable {
 interface State {
   data: DataFrame[];
   transformations: TransformationsEditorTransformation[];
+  search?: string;
```

Comment:
nit: defaulting this to an empty string rather than `undefined` might make the code easier to reason with.